src/components/Components/CandidateSection/CandidateSection.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import "./CandidateSection.scss"
+import ViewMoreIcon from "./../../icons/Vector.png"
+
+function CandidateSection() {
+  return (
+    <div className='CandidateSection'>
+        <div className='CandidateSection__wrapper'>
+            <h2 className='CandidateSection__wrapper__title'>
+                Meet Candidates
+                <div className='CandidateSection__wrapper__title__rec'></div>
+            </h2>
+            <div className='CandidateSection__wrapper__viewmore'>
+              <a href='#' className='CandidateSection__wrapper__viewmore__text'>
+                  View More
+                  <img className='CandidateSection__wrapper__viewmore__icon' src={ViewMoreIcon} />
```

Comment:
need some spacing to the image
again  remove the underline thingy done by links

Reply:
@jasper-prgh Jasper, what is the benefit of creating 2 elements for viewmore--bottom and displaying them depending on the screen size? I mean, isn't it cleaner just to change the position of viewmore--bottom to absolute, depending on the screen size (for mobile) and move it to the bottom? 

here what I mean https://vdb99b.csb.app/
https://codesandbox.io/s/meet-candidates-vdb99b?file=/src/styles.css